charts/pravega/templates/pravega.yaml

```diff
@@ -111,6 +111,10 @@ spec:
       repository: {{ .Values.image.repository }}
       pullPolicy: {{ .Values.image.pullPolicy }}
     controllerReplicas: {{ .Values.controller.replicas }}
+    {{- if .Values.segmentStore.initContainers }}
+    authImplementations:
+{{ toYaml .Values.authImplementations | indent 6 }}
```

Comment:
I'm not following the logic why the authImplementations is only added in if segmentStore.initContainers is not empty.

Reply:
it was copy/paste error. Fixed that